attributes/_install.rb

```diff
@@ -39,13 +39,3 @@
       '-x86-release.msi'
     end
   end
-
-default['splunk']['package']['provider'] =
```

Comment:
Don't forget to remove this from documentation

Reply:
removed in 3ee2707